app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<p id="notice"> <%= notice %></p>
+
+<p>
+  <strong>Title:</strong>
+  <%= @task.title %>
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @task.description %>
+</p>
+
+<p>
+  <strong>Project:</strong>
+  <%= link_to @task.project.name, url_for(@task.project)%>
+</p>
+
+<p>
+  <strong>Deadline:</strong>
+  <%= @task.deadline_at %>
+</p>
+
+<%= link_to 'Edit', edit_task_path(@task) %>
+<%= link_to 'Back', tasks_path %>
```

Comment:
It would be convenient to see destroy action here.

Reply:
It will be done